orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/pipeline/PipelineStage.java

```diff
@@ -43,8 +43,12 @@
   public static final String PIPELINE_CONFIG_TYPE =
       StageDefinitionBuilder.getType(PipelineStage.class);
 
-  @Autowired(required = false)
-  ExecutionRepository executionRepository;
+  final ExecutionRepository executionRepository;
+
+  @Autowired
+  public PipelineStage(ExecutionRepository executionRepository) {
```

Comment:
This _appears_ to be a breaking change. I don't know when we wouldn't have an `ExecutionRepository`, but in this change, we're making it required, whereas in the past it was optional.

Reply:
ah, good call. Not that this stage will work without a repo though.... not sure what to do. I think making it optional like it was before will give a wrong impression that we can work without it. and making the whole stage/bean conditional on repo seems weird, what do you think?

Reply:
Yeah - I'd say the change you have is correct. I didn't take a look myself, so it just jumped out at me as something that _could_ break. If the stage won't work without it, you're just fixing things. :)